core/src/main/java/ai/timefold/solver/core/impl/score/stream/collector/connectedRanges/ConnectedRangeImpl.java

```diff
@@ -1,24 +1,29 @@
-package ai.timefold.solver.examples.common.experimental.impl;
+package ai.timefold.solver.core.impl.score.stream.collector.connectedRanges;
```

Comment:
I'd go for just `ranges` - we generally don't do camel case in package names.

Reply:
`ranges` is too vague I think. `connected_ranges` ? (from `testdata/domain/allows_unassigned` and friends, which I misremberred as being `camelCase` instead of `snake_case`).

Reply:
OK.